Casks/graalvm-ce-java11.rb

```diff
@@ -29,6 +29,11 @@
     Installing GraalVM CE (Java 11) in #{JVMS_DIR} requires root permissions.
     You may be asked to enter your password to proceed.
 
+    The first time you run GraalVM CE on macOS Catalina, you may get a warning
```

Comment:
It's also on upgrades (`brew cask upgrade`), or at least the `xattr` way requires running it multiple times (does clicking the button run `xattr`? 🤷‍♂).